apps/deep-clone/eslint.config.js

```diff
@@ -28,14 +30,12 @@ export default [
       'react-hooks': reactHooks,
     },
     rules: {
+      ...typescript.configs['flat/recommended'].rules,
       ...react.configs.recommended.rules,
       ...reactHooks.configs.recommended.rules,
       'react/react-in-jsx-scope': 'off',
       'react/no-unescaped-entities': 'off',
-      'no-unused-vars': [
-        'warn',
-        { varsIgnorePattern: '^_', argsIgnorePattern: '^_', caughtErrorsIgnorePattern: '^_' },
-      ],
+      'no-unused-vars': 'off',
```

Comment:
💭  I don't think we should remove this, but if you want we can fix it after successfully migrating the app so we don't add unnecessary noise